app/controllers/partners/partner_external_accounts_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Partners
+  class PartnerExternalAccountsController < ApplicationController
+    rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+    def pundit_user
+      current_partner
+    end
+
+    def destroy
+      @partner_external_account = PartnerExternalAccount.find(params[:id])
+      authorize @partner_external_account.partner
```

Comment:
:memo: This bugged me. I expected to have `current_partner.partner_external_account.find` to ensure we wouldn't even retrieve a record that's not the one form the `current_partner`. In the end, `authorize` does the job :+1:

Reply:
AH, je savais pas qu'on pouvait faire un find sur une collection, je vais faire ça

Reply:
Yes, et tu peux regarder aussi dans le genre tout ce qui est [`create_with`](https://devdocs.io/rails~6.1/activerecord/querymethods#method-i-create_with) et [`find_or_initialize_by`](https://devdocs.io/rails~6.1/activerecord/relation#method-i-find_or_initialize_by) c'est assez pratique pour un autre truc que j'ai vu dans la PR.